spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+RSpec.describe 'User integrations', type: :feature do
+  it 'displays user profile in show page' do
+    user = create(:user)
+    visit user_path(user)
+    expect(page).to have_content('Andrew Carnegie')
+  end
```

Comment:
- [ ] [REQUIRED] There is supposed to be a test that asserts you can see the user's profile picture.
- While it's good that the test verifies some content on the page related to the user, such as the user's name ('Andrew Carnegie'), it does not specifically check for the presence of the profile picture.
- Kindly add a test that asserts you can see the user's profile picture.